File: Data Structures & Algorithms/daily-temperatures/submission-0.java
class Solution {
    public int[] dailyTemperatures(int[] t) {
        int[] res = new int[t.length];
        for(int i=0; i<t.length; i++)
        {
            int cur = 1;
            for(int j=i+1; j<t.length; j++)
            {
                if(t[j]>t[i])
                {
                    res[i] = cur;
                    break;
                }
                else
                {
                    cur += 1;
                }
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/generate-parentheses/submission-0.py
class Solution:
    def generateParenthesis(self, n):
        if n < 1:
            return []
        if n == 1:
            return ["()"]

        def isValid(s):
            if not s:
                return True
            if s[0] == ")":
                return False
            st = []
            for i in s:
                if i == "(":
                    st.append(i)
                else:
                    if not st or st.pop() != "(":
                        return False
            if not st:
                return True
            return False

        res = []
        def backtrack(cur):
            if len(cur) == 2*n and isValid(cur):
                res.append(cur[::])
                return
            if len(cur) > 2*n:
                return
            l = cur+"("
            backtrack(l)
            r = cur+")"
            backtrack(r)

        backtrack("")
        
        return res